manifests/params.pp

```diff
@@ -3,7 +3,6 @@
 # This class is meant to be called from consul
 # It sets variables according to platform
 #
-# @api private
```

Comment:
Accidental removal?

Reply:
no. the params.pp doesn't have assert_private() (and adding it will break it). That comment was added by accident. I cleaned it up.

Reply:
It still is a private class as users aren't supposed to call it AFAIK. I'd think it should stay so `puppet-strings` doesn't document it.

Reply:
fair point. Added it back.